src/components/breadcrumb/Breadcrumb.component.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactNode } from "react";
+import "styles/components/Breadcrumb.scss";
+import {action} from "@storybook/addon-actions";
+import Breadcrumb from 'react-bootstrap/Breadcrumb';
+
+export type BreadcrumbItem = {
+    title: JSX.Element | string;
+    path: string;
+    isActive?: boolean;
+}
+export type BreadcrumbItems = {
+    breadcrumbItems:Array<BreadcrumbItem>;
+    className?:string;
```

Comment:
unused prop

Reply:
Its used in case of Breadcrumbs function.